lib/chat/chat_messenger.dart

```diff
@@ -109,44 +135,56 @@ class ChatMessengerState extends State<ChatMessenger> {
               maxLines: null, // this allows for multi-line input
               textInputAction: TextInputAction.send,
               controller: textInputController,
-              onSubmitted: submitMessage,
+              onSubmitted: (value) => submitMessage(0, value),
             )
           ),
           IconButton(
             icon: Icon(Icons.send),
             iconSize: 25.0,
             color: Theme.of(context).primaryColor,
-            onPressed: () => submitMessage(textInputController.value.text),
+            onPressed: () => submitMessage(0, textInputController.value.text),
```

Comment:
Same thing here with the `0` type. It could be defined as a constant and imported for readability.